server/src/main/java/org/apache/druid/segment/realtime/appenderator/AppenderatorImpl.java

```diff
@@ -247,6 +249,18 @@ public class AppenderatorImpl implements Appenderator
     } else {
       log.debug("Running closed segments appenderator");
     }
+
+    try {
+      String pThreads = System.getProperty("druid.exp.persist.threads");
```

Comment:
I think this property should make its way to `docs/` as well.

Reply:
Will add it

Reply:
This was an unused property I had added it as an earlier trial, cleaned it up. Also added documentation for the actual configuration setting @jasonk000 @abhishekagarwal87 @gianm (https://apachedruidworkspace.slack.com/archives/C030CMF6B70/p1683280882330109)

Reply:
Any comment @jasonk000